### PR TITLE
Integrate burner-core for reading balances

### PR DESCRIPTION
**Pull request by dmihal**
This PR adds the modules @burner-wallet/core and @burner-wallet/assets to the project (code available in [austintgriffith/burner-core](https://github.com/austintgriffith/burner-core)). The burner core object is constructed and used for querying account balances.

This is the first step in the iterative process of splitting code between the burner-wallet and burner-core repos.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@burner-wallet/assets": "^0.0.1",
+    "@burner-wallet/core": "^0.0.1",
```
[the remark this answers is not in the document]
Comment (dmihal) [reply package.json]: Done :)

**Comment by dmihal** [comment]
Made some changes to burner-core and updated it to 0.0.4.

I've tested in chrome, firefox and safari, with and without metamask. Everything seems to be working well for me now. Let me know if you're still seeing those issues on your Chrome and I'll investigate more.

**Comment by dmihal** [comment]
I see, so is the issue that 0.005 ETH should be rounded to 0.01 ETH? Or should it be just displayed as 0.005 ETH.

I chose to use toFixed since the numbers we're using may be too large to fit in Javascript numbers (that's why Web3 returns them as strings too). Although they're probably a small enough size once we convert them to ETH, so I could switch to toFixed if you think it's better.

**Comment by TimDaub** [comment]
I believe that getDisplayBalance should only cut those decimals that have a
value of < $0.01. Or don't cut at all. Otherwise users will get a wrong
balance even thought they have ETH deposited.

If I may ask: What's the reason for cutting decimals? Burner always
displays ETH balance in Dollars, so it shouldn't be necessary, no?

**Comment by dmihal** [comment]
Typically the burner-core code will handle the conversion to USD, however in this PR, we're still letting burner-wallet handle that.

Here's the best options I see:
1. Just increase the number of decimal points, since burner-wallet will be handling them anyways, so getDisplayBalance(address, 20). That should give us enough data to make the conversions correctly.
2. Use `getUSDBalance` instead of `getDisplayBalance` (the function isn't written but would be a quick addition).

I think eventually we want to do #2, since I think conversion to USD is something we can delegate to the core. However, in the interest of keeping this PR small and having minimal impact on the existing codebase, I'd suggest we do #1 for now.

I've added a commit doing #1, but let me know if you have any thoughts or think we should be switching to #2 now.

**Comment by TimDaub** [comment]
> However, in the interest of keeping this PR small and having minimal impact on the existing codebase, I'd suggest we do #1 for now.

Agreed. 

I tested it and it works now. Thx!

**Pull request merged**